tests/e2e/hooks.ts

```diff
@@ -0,0 +1,29 @@
+import { Before, BeforeAll, After, AfterAll, setDefaultTimeout } from '@cucumber/cucumber'
+import { chromium } from '@playwright/test'
+import config from './config'
+import { deleteAllFiles, emptyTrashbin } from './utils/helpers'
+
+setDefaultTimeout(config.timeout * 1000)
+
+BeforeAll(async function (): Promise<void> {
+  global.browser = await chromium.launch({
+    slowMo: config.slowMo,
+    headless: config.headless,
+    channel: 'chrome'
+  })
```

Comment:
thanks, let's make sure we keep it consistent with other oCIS extensions (at least the one's we're developing...)
@amrita-shrestha is the code example above from [oCIS web](https://github.com/owncloud/web)? If so, would you mind to raise this issue also for [web-app-dicom-viewer](https://github.com/owncloud/web-app-dicom-viewer/) and [web-app-presentation-viewer](https://github.com/JankariTech/web-app-presentation-viewer)?

Reply:
I think i am not eligible to charge time for those owncloud extension project but I messaged @salipa and @saw-jan who are in charge of those project

Reply:
@amrita-shrestha you're very welcome to review tests in dicom viewer and open issues for things where you think the quality of tests needs to be improved. Time spent on this can be charged to the project, happy to give you the details.

Reply:
opened an issue for the suggestion about support of additional browsers, this improvement will be addressed in a separate PR